WordPress/gutenberg
# Editor Onboarding: "The Basics" help section - Details Part 2

## Related PRs
* https://github.com/wordpress-mobile/gutenberg-mobile/pull/3824
* https://github.com/wordpress-mobile/WordPress-iOS/pull/17008
* https://github.com/wordpress-mobile/WordPress-Android/pull/15164

## Description
Fixes https://github.com/WordPress/gutenberg/issues/30626. Iterates upon the work completed in https://github.com/WordPress/gutenberg/pull/33790. This work improves accessibility, copy, and design of the editor help section. 

## How has this been tested?

⚠️ The Editor Onboarding features, including the Help button, are only available to 50% of users based on user ID. You must be logged into a WPCom account or receive a anonymous ID that is included in the cohort. 

<details><summary>Help Section: Visual</summary>

1. Install builds for iOS and Android from the sibling PRs. 
2. Launch the post editor. 
3. Tap the three-dot menu in the top right corner. 
4. Tap Help. 
5. ℹ️ **Expected:** Navigating to each topic results in the correct content is displayed. 
6. Close the editor. 
7. Enable dark mode of the device.
8. Repeat steps 2-5. 

</details>

<details><summary>Help Section: Audible</summary>

1. Install builds for iOS and Android from the sibling PRs. 
2. Launch the post editor. 
3. Enable VoiceOver/TalkBack. 
4. Select and activate the three-dot menu in the top right corner. 
5. Select and activate Help.
6. ℹ️ **Expected:** Navigating and comprehending each topic is easy with the screen reader. 

</details>

<details><summary>Existing Bottom Sheet Headers</summary>

1. Install builds for iOS and Android from the sibling PRs. 
2. Launch the post editor. 
3. Add a Paragraph block. 
4. Tap Link button in toolbar. 
5. Tap Link to Cell. 
6. ℹ️ **Expected:** Verify the Cancel/X button, title, and Apply/Checkmark button all render and function as expected. 
7. Add a Cover block. 
8. Add background media. 
9. Tap block settings button. 
10. Tap Edit focal point. 
11. ℹ️ **Expected:** Verify the Cancel/X button, title, and Apply/Checkmark button all render and function as expected. 

</details>

## Screenshots 

<details><summary>UX Interactions</summary>

https://user-images.githubusercontent.com/438664/130818291-58a4a8d0-43f4-457a-9dd9-8c1315143a1c.MP4

</details>


<details><summary>Screenshots</summary>

| iOS | Android |
| --- | --- |
| ![help-index-ios](https://user-images.githubusercontent.com/438664/130817252-b544f998-d922-412c-8ed4-7141dedc3c44.PNG) | ![help-index-android](https://user-images.githubusercontent.com/438664/130817300-039a6a52-cad3-47a5-8f12-6fe2a63777dc.jpg) |
| ![help-index-ios-dark](https://user-images.githubusercontent.com/438664/130817350-134861b7-33bf-42e2-9d34-101430c89aed.PNG) | ![help-index-android-dark](https://user-images.githubusercontent.com/438664/130817394-b3c28da5-0d3f-4598-a2ca-6e190ed4d154.jpg) |
| ![help-detail-ios](https://user-images.githubusercontent.com/438664/130817437-81375244-ac97-4812-a8f5-4bc2617cd8d1.PNG) | ![help-detail-android](https://user-images.githubusercontent.com/438664/130817474-9ed5d892-197f-46ed-8b96-de5e24562237.jpg) |
| ![help-detail-ios-dark](https://user-images.githubusercontent.com/438664/130817499-c259cacc-d122-4c4f-b844-df1b76923656.PNG) | ![help-detail-android-dark](https://user-images.githubusercontent.com/438664/130817527-bdd883bc-69cc-4e54-967e-588aa7d82781.jpg) |

</details>

## Types of changes
New feature

## Checklist:
